charters/DocumentationCharter.md

```diff
@@ -0,0 +1,54 @@
+# Docs Working Group Charter (Proposed)
+
+## Mission and Goals
+
+The Docs Working Group serves as a support, helper, and advisory body to Jupyter subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community
```

Comment:
It might be helpful to use a term or terms that are more specific that "make". Do we mean author? Maintain? I am trying to get a sense of the scope of the WGs work.

Reply:
Maybe some of this detail is (already) in the activities below.